app/src/main/AndroidManifest.xml

```diff
@@ -94,6 +94,7 @@
 
         <activity
             android:name=".auth.SignupActivity"
+          android:configChanges="orientation|screenLayout|screenSize"
```

Comment:
Hi, Please fix the formatting changes over here. Happy to merge post this

Reply:
I am beginner to open source contribution I does not  understand what  formatting changes make over above ?